src/util/encryption.ts

```diff
@@ -0,0 +1,13 @@
+import { createHmac } from "crypto";
+import { envVars } from "../config/env";
+
+const getSharedSecret = (): string => {
+	return envVars.HASH_SECRET;
+}
```

Comment:
do you really need a function if it's just a constant?

Reply:
I accept.